acceptance/config/nodes/debian-7-passenger.yaml

```diff
@@ -0,0 +1,19 @@
+HOSTS:
+  master:
+    roles:
+      - master
+    platform: debian-wheezy-x86_64
+    hypervisor: vcloud
+    template: debian-7-x86_64
+  agent:
+    roles:
+      - agent
+    platform: el-7-x86_64
+    hypervisor: vcloud
+    template: redhat-7-x86_64
+CONFIG:
+  datastore: instance0
+  resourcepool: delivery/Quality Assurance/FOSS/Dynamic
+  folder: Delivery/Quality Assurance/FOSS/Dynamic
+  pooling_api: http://vmpooler.delivery.puppetlabs.net/
+  is_puppetserver: false
```

Comment:
We need separate host config files for passenger because we don't test el6, debian, or ubuntu on the master, only on the agent side (during normal AIO acceptance).